metagraph/src/annotation/binary_matrix/base/binary_matrix.cpp

```diff
@@ -32,6 +34,25 @@ BinaryMatrix::slice_rows(const std::vector<Row> &row_ids) const {
     return slice;
 }
 
+void BinaryMatrix::slice_columns(const std::vector<Column> &column_ids,
+                                 const ColumnCallback &callback) const {
+    size_t nrows = num_rows();
+
+    #pragma omp taskloop
+    for (size_t k = 0; k < column_ids.size(); ++k) {
+        Column j = column_ids[k];
+        auto column = get_column(j);
+        size_t nsetbits = column.size();
+        callback(j, bit_vector_smart([s=std::move(column)](const auto &index_callback) {
+            for (uint64_t i : s) {
+                index_callback(i);
+            }
+        }, nrows, nsetbits));
```

Comment:
What happened to that other bitmap based on a sorted vector? Was it in this PR?

Reply:
I used that for the BRWT implementation, but forgot to change the general one to also use it.